daemon/heartbeat.c

```diff
@@ -996,8 +1042,12 @@ send_hb()
         {
             if (index != _my_index)
             {
-                sendto(hbvar.socket, &pkt, sizeof(pkt), 0,
-                       &hbvar.sa_to[index], sizeof(hbvar.sa_to[index]));
+                const int ret = sendto(hbvar.socket, &pkt, sizeof(pkt), 0,
+                    &hbvar.sa_to[index].sa, sizeof hbvar.sa_to[index]);
```

Comment:
Passing `sizeof hbvar.sa_to[index]` is possibly to large for the intended address. While this probably doesn't matter in practice, it would be preferable to pass the correct length.